assets/js/modules/analytics-4/components/audience-segmentation/settings/SettingsCardVisitorGroups/SetupSuccess/SetupSuccessContent.js

```diff
@@ -29,15 +29,15 @@ import { addQueryArgs } from '@wordpress/url';
 import { AREA_MAIN_DASHBOARD_TRAFFIC_AUDIENCE_SEGMENTATION } from '../../../../../../../googlesitekit/widgets/default-areas';
 import { CORE_LOCATION } from '../../../../../../../googlesitekit/datastore/location/constants';
 import { CORE_SITE } from '../../../../../../../googlesitekit/datastore/site/constants';
-import { CORE_USER } from '../../../../../../../googlesitekit/datastore/user/constants';
+import { CORE_UI } from '../../../../../../../googlesitekit/datastore/ui/constants';
 import { trackEvent } from '../../../../../../../util';
 import useViewContext from '../../../../../../../hooks/useViewContext';
 import { useDispatch, useSelect } from 'googlesitekit-data';
 import { Button } from 'googlesitekit-components';
 import CheckFill from '../../../../../../../../svg/icons/check-fill.svg';
 
-export const SETTINGS_VISITOR_GROUPS_SETUP_SUCCESS_NOTIFICATION =
-	'settings_visitor_groups_setup_success_notification';
+export const SHOW_SETTINGS_VISITOR_GROUPS_SUCCESS_NOTIFICATION =
+	'showSetupSuccess';
```

Comment:
Let's set the value to be something more unique to avoid potential future conflicts.

```suggestion
export const SHOW_SETTINGS_VISITOR_GROUPS_SUCCESS_NOTIFICATION =
	'show_settings_visitor_groups_success_notification';
```